OmsAgent/watcherutil.py

```diff
@@ -104,7 +104,10 @@ def create_telemetry_event(self, operation, operation_success, message, duration
     		]
             }}"""
 
-	return template.format(operation, operation_success, message, duration)
+        operation_success_as_string = str(operation_success).lower()
+        formatted_message = message.replace("\n", "\\n").replace("\t", "\\t").replace('"', '\\"')
```

Comment:
Shouldn't " be replaced with \" instead of \\" ?

Reply:
Ok will submit in another PR